### PR TITLE
Add game data class, replace game_id.

### PR DESCRIPTION
**Pull request by khazhyk**
it works :+1:

### CODE REVIEW

File: discord/member.py [hunk 1]
```diff
@@ -75,7 +76,10 @@ def __init__(self, deaf, joined_at, user, roles, mute, **kwargs):
         self.joined_at = parse_time(joined_at)
         self.roles = roles
         self.status = Status.offline
-        self.game_id = kwargs.get('game_id', None)
+        game = kwargs.get('game', None)
+        if game is not None:
+            game = Game(**game)
+        self.game = game
```
Comment (SamSchlegel) [inline discord/member.py]: this can just be `self.game = game and Game(**game)`

File: discord/client.py [hunk 2]
```diff
@@ -1427,27 +1426,27 @@ def change_status(self, game_id=None, idle=False):
...
 
+        if game is not None and not isinstance(game, Game):
+            raise InvalidArgument('game must be of Game or None')
+
         idle_since = None if idle == False else int(time.time() * 1000)
-        try:
-            game_id = None if game_id is None else int(game_id)
-        except:
-            raise InvalidArgument('game_id must be convertible to an integer or None')
+        game = None if game is None else {'name': game.name}
```
Comment (SamSchlegel) [inline discord/client.py]: Can just be `game = game and {'name': game.name}`

File: discord/member.py (hunk 1 above)
Comment (jhgg) [inline discord/member.py]: .get() returns None by default, no need to add it explicitly.

**Comment by khazhyk** [comment]
Note: change_status will not send a respective presence update for yourself. As is, your own member object will not be updated with your game status. Not sure if this was the old behavior or not.

**Comment by ericburns** [comment]
:+1:

**Pull request merged**